README.md

```diff
@@ -66,8 +69,8 @@ async def my_handler(keys: List[str], datums: Iterator[Datum], md: Metadata) ->
 
 
 if __name__ == "__main__":
-    grpc_server = UserDefinedFunctionServicer(reduce_handler=my_handler)
-    asyncio.run(grpc_server.start_async())
+    grpc_server = AsyncServer(reduce_handler=my_handler)
+    asyncio.run(grpc_server.start())
```

Comment:
lets use aiorun for asyncio for lifecycle support

Reply:
Done